content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -16,8 +16,23 @@ be used when running Kubernetes on Amazon Web Services.
 
 ### Node Name
 
+#### Default
+
 The AWS cloud provider uses the private DNS name of the AWS instance as the name of the Kubernetes Node object.
 
+#### Explicit
```

Comment:
indicate this requires all components (kubelet, api-server, and kube-controller-manager) to be at 1.12 or higher